CONVERT_SERVICE/convert_service/convert_video/util/utilities.py

```diff
@@ -0,0 +1,34 @@
+#
+# @Utilities.py Copyright (c) 2021 Jalasoft.
```

Comment:
Little bit detail: Utilities.py should be with lower case

Reply:
Corrected, thanks!